app/assets/templates/directives/buddy_request_box.html

```diff
@@ -0,0 +1,17 @@
+<div id="buddyRequestInfoBox"></div>
+<div class="panel panel-info" ng-controller="BuddyNotificationsCtrl as buddyCtrl">
```

Comment:
I meant you could define the controller in the directive definition, not moving the `ng-controller` to the directive